Book API Homework/README.md

```diff
@@ -0,0 +1,26 @@
+# Book API
+This folder contains an API and an virtualenv.
+
+To copy this branch and work with the API, please follow the steps below:
+
+* Create a folder to copy the branch.
+* Clone the repository: `git clone https://github.com/bounswe2017group7.git`
```

Comment:
Repo link is broken. 
`git clone https://github.com/bounswe/bounswe2017group7.git`